app/layout.tsx

```diff
@@ -1,6 +1,39 @@
-export const metadata = {
-  title: 'Next.js',
-  description: 'Generated by Next.js',
+import { Background } from '@/components/background'
+import type { Metadata } from 'next'
+
+import './globals.css'
+
+export const metadata: Metadata = {
+  title: 'Monad Developer MCP - AI-Powered Development Tools',
+  description:
+    'Set up Monad Developer MCP in your IDE to access real-time documentation, query accounts, analyze transactions, and generate CPI statements. Built for the Monad developer community',
+  keywords: [
+    'Monad',
+    'blockchain development',
+    'MCP',
+    'developer tools',
+    'smart contracts',
+    'web3',
+  ],
+  openGraph: {
+    title: 'Monad Developer MCP - AI-Powered Development Tools',
+    description:
+      'Access Monad blockchain development tools directly in your IDE with real-time documentation and program analysis capabilities.',
+    type: 'website',
+    url: 'https://monad-mcp-tau.vercel.app',
+    siteName: 'Monad Developer MCP',
+    images: [
+      {
+        url: 'https://mcp.Monad.com/og-image.png',
```

Comment:
One more here

Reply:
Actually there is no such image, I will remove this entry